### PR TITLE
Add initial Hearings view with tabs for admins

### PR DESCRIPTION
**Pull request by tuomas777**
The view contains published hearings, hearing publishing queue and
draft hearings. It isn't added to any route yet.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,6 +53,7 @@
     "react-redux": "^4.0.0",
     "react-router": "^1.0.0-rc3",
     "react-router-bootstrap": "^0.19.3",
+    "react-tabs": "^0.5.5",
```
Comment (akx) [inline package.json]: Why not [react-bootstrap's tabs](http://react-bootstrap.github.io/components.html#tabs)?

File: src/views/AdminHearings.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
+import React from 'react';
```
Comment (akx) [inline src/views/AdminHearings.js]: Maybe admin views should live in `views/admin/...`?

File: package.json (hunk 1 above)
Comment (JsseL) [inline package.json]: Yep, I think we should stick with bootstrap components if possible for consistency.

**Comment by tuomas777** [comment]
Changed to use react-bootstrap's tabs, and relocated the view file as suggested.

File: src/views/admin/AdminHearings.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  componentDidMount() {
+    const {dispatch} = this.props;
+    AdminHearings.fetchData(dispatch);
+  }
+
+  render() {
+    const {formatMessage} = this.props.intl;
+    const {hearingLists} = this.props;
+
+    return (<div className="container">
```
Comment (JsseL) [inline src/views/admin/AdminHearings.js]: Put the `div` into its own line. Makes this look more like properly intended HTML.

**Comment by JsseL** [comment]
Looks alright. I think we can wery well merge this and implement the actions etc. afterwards.

**Comment by JsseL** [comment]
Uncommenting `@import "@{bootstrap-dir}/component-animations";` line in `app.less` might fix the animation issue.

File: src/views/admin/AdminHearings.js [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+import Row from 'react-bootstrap/lib/Row';
+import {Tab, Tabs} from 'react-bootstrap';
+
+class AdminHearings extends React.Component {
+  static fetchData(dispatch) {
+    const now = new Date().toISOString();
+    return Promise.all([
+      dispatch(fetchHearingList("publishedHearings", "/v1/hearing/", {published: "True", open_at_lte: now})),
+      dispatch(fetchHearingList("publishingQueueHearings", "/v1/hearing/", {published: "True", open_at_gt: now})),
+      dispatch(fetchHearingList("draftHearings", "/v1/hearing/", {published: "False"}))
```
Comment (akx) [inline src/views/admin/AdminHearings.js]: Add dangling comma.

File: src/views/admin/AdminHearings.js [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+        </Tab>
+      </Tabs>
+    </div>);
+  }
+}
+
+AdminHearings.propTypes = {
+  intl: intlShape.isRequired,
+  dispatch: React.PropTypes.func,
+  hearingLists: React.PropTypes.object
```
Comment (akx) [inline src/views/admin/AdminHearings.js]: Add dangling comma.

**Pull request merged**